### DISCUSSION TITLE
Connect to host from docker compose service.

### DISCUSSION THREAD

**Discussion by malylemireflo** [community/community#70257] [Actions]
### Select Topic Area

Bug

### Body

I have a dapr service running in docker compose that need to connect to a host api while running integration tests.

I'm using the dapr --app-channel-address to map it to host.docker.internal

Everything works fine locally with Docker Desktop for windows. But I'm not able to make it work in github actions.
My github action run with ubuntu so I've added this to my docker file.
extra_hosts:  
  - "host.docker.internal:host-gateway"
Also tried using 172.17.0.1 directly.

But I still cannot connect from dapr inside docker to the host. Is there another way since I know you have a custom network for github actions?

**Comment by malylemireflo** [comment]
Anyone can help? Everywhere I look it seems that contacting the host from a docker compose service is not working in a github action runner.

**Comment by malylemireflo** [comment]
It's ok found that my api also need to be on http://host-gateway:port

**Comment by caarmen** [comment]
I just ran into this (using just docker, not docker compose). 

Let me just share a bit more details about the fix (indeed it's about host-gateway), in case it helps anybody:

I had to adapt my `docker run` command: I had to add `--add-host=host.docker.internal:host-gateway` to it.

Found the information in this blog: https://medium.com/@TimvanBaarsen/how-to-connect-to-the-docker-host-from-inside-a-docker-container-112b4c71bc66
